test/index.ts

```diff
@@ -21,3 +27,34 @@ describe('Access the server', () => {
     expect(response.body.data.hello).to.be.eq('Hello, world!');
   });
 });
+
+describe('Mutation createUser', () => {
+  it('Run mutation createUser', async () => {
```

Comment:
costumamos montar os its para descrever cenarios de teste e não cada item a ser avaliado em especifico,
coisas como:
- deve ser possivel criar um usuario: e ai testa a mutation, valida o usuario no banco e tudo mais no mesmo it
- deve fazer o hash da senha
- cenários de erro (se eu não me engano, a essa altura o email era para ser chave unica, com isso nao deve ser possivel criar dois usuarios com o mesmo email, certo? E ai seria interessante testar isso)

Reply:
Adicionei o teste de email duplicado

Reply:
Também movi o teste  do usuário no banco pro mesmo `it` que roda a mutation pra ficar de acordo com o comentado